tests/unit/libs/fetcher.client.js

```diff
@@ -86,7 +87,7 @@ describe('Client Fetcher', function () {
                     },
                     post : function (url, headers, body, config, done) {
                         expect(url).to.not.be.empty;
-                        expect(url).to.equal(corsPath + '?_csrf=' + context._csrf);
+                        expect(url).to.equal(corsPath + '?_csrf=' + context._csrf + '&timeout=3000');
```

Comment:
yeah, it seems we don't need this feature anymore, let me remove the timeout part